### PR TITLE
feat(EditProject): ✨ added contributors page

### PR DESCRIPTION
**Pull request by bbtgnn**

### CODE REVIEW

File: pages/project/[id]/edit/contributors.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  contributors: ContributorsStepValues;
+}
+
+const EditContributors: NextPageWithLayout = () => {
+  const project = useProject();
+
+  /* Form setup */
+
+  const defaultValues: EditContributorsValues = {
+    contributors: project.metadata.contributors || [],
```
Comment (puria) [inline pages/project/[id]/edit/contributors.tsx:30]: Seems that an empty array/list is not the same as undefined on the Zenflows backend, can you double check this?

**Review by puria — CHANGES_REQUESTED** [review]

File: pages/project/[id]/edit/contributors.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+// Components
+import EditProjectLayout from "components/layout/EditProjectLayout";
+import FetchProjectLayout, { useProject } from "components/layout/FetchProjectLayout";
+import Layout from "components/layout/Layout";
+import EditFormLayout from "components/partials/project/edit/EditFormLayout";
+
+//
+
+export interface EditContributorsValues {
```
[the remark this answers is not in the document]
Comment (bbtgnn) [reply pages/project/[id]/edit/contributors.tsx:20]: No actually it's just a habit of exporting interfaces 😬

File: pages/project/[id]/edit/contributors.tsx (hunk 1 above)
Comment (bbtgnn) [reply pages/project/[id]/edit/contributors.tsx:30]: Yes, it's true. I'll fix this

Comment (bbtgnn) [reply pages/project/[id]/edit/contributors.tsx:31]: It works also with empty arrays now :)

**Review by puria — APPROVED** [review]

**Pull request merged**